client/locations-list-map.js

```diff
@@ -1130,9 +1135,14 @@ function initializeEmbedLocationCollapse() {
 }
 
 function refreshList(data, filters) {
+  if (!showList) {
```

Comment:
do we need the `showList` conditional here *and* where the function is called? seems redundant but I might be missing something

Reply:
I debated with myself about that. Decided to call the redundancy "defense in depth". I'll change it.